src/collections/integrations/jenkins-operator/index.mdx

```diff
@@ -18,7 +18,7 @@ howItWorksDetails: "Collaboratively manage infrastructure with your coworkers sy
 published: true
 ---
 <p>
-    CNCF Jaeger, a Distributed Tracing Platform
+    A native Kubernetes operator
```

Comment:
@Shivam-AfA this is non-descriptive as this could be any one of hundreds of Kubernetes operators.

Reply:
**A Kubernetes native operator which manages operations for Jenkins on Kubernetes, built with immutability and declarative Configuration as Code in mind.**